azurerm/internal/services/network/point_to_site_vpn_gateway_resource.go

```diff
@@ -93,6 +93,48 @@ func resourceArmPointToSiteVPNGateway() *schema.Resource {
 								},
 							},
 						},
+
+						"route_config": {
```

Comment:
can we drop config from here? 
```suggestion
						"route": {
```